### ISSUE TITLE
Installing zram on breath

### ISSUE THREAD

**Issue by nicro87** [cb-linux/breath#209]
First let me say how amazing your scripts work. For example Kodi on my lenovo 100e is now so much faster! Thanks!

But again with Kodi I sometimes have the issue that it crashes when there is a lot todo. I think it's because the RAM is gettin full.
Is there a option to install something like swap on ubuntu?
I tried with zram-config but this das not work because what I unterstand from an earlier issue that swap is not enabled in this kernel. 

Can I get around this? Has someone an idea to get this working?

Thanks,
Nico

**Comment by MilkyDeveloper** [comment]
Yeah, swap doesn't work with the ChromeOS kernel. I believe `zram` doesn't depend on swap, though. I haven't tested `zram` on my Chromebook because it has 8gb of RAM, but [here](https://github.com/cb-linux/breath/issues/204#issuecomment-1133802766) are the instructions to enable it.

I think `zram-config` should work.

**Comment by nicro87** [comment]
Thanks. I try this tomorrow and report if it works.

**Comment by nicro87** [comment]
Works. Thank you!
